rotkehlchen/tests/unit/decoders/test_aave.py

```diff
@@ -216,3 +223,668 @@ def test_aave_eth_withdraw_v1(database, ethereum_inquirer):
+                topics=[
+                    hexstring_to_bytes('0xc6a898309e823ee50bac64e45ca8adba6690e99e7841c45d754e2a38e9019d9b'),  # noqa: E501
+                    hexstring_to_bytes('0x000000000000000000000000408e41876cccdc0f92210600ef50372656052a38'),  # noqa: E501
+                    hexstring_to_bytes('0x00000000000000000000000000000000000cd56832ce5dfbcbff02e7ec639bc9'),  # noqa: E501
+                    hexstring_to_bytes('0x0000000000000000000000000000000000000000000000000000000000000000'),  # noqa: E501
+                ],
+            ),
+        ],
+    )
+    dbevmtx = DBEvmTx(database)
+    with dbevmtx.db.user_write() as cursor:
+        dbevmtx.add_evm_transactions(cursor, [transaction], relevant_address=None)
+    decoder = EthereumTransactionDecoder(
+        database=database,
+        ethereum_inquirer=ethereum_inquirer,
+        transactions=eth_transactions,
+    )
+    with dbevmtx.db.user_write() as cursor:
+        events = decoder.decode_transaction(cursor, transaction=transaction, tx_receipt=receipt)
+    expected_events = [
+        HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=0,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=ZERO, usd_value=ZERO),
+            location_label='0x00000000000Cd56832cE5dfBcBFf02e7eC639BC9',
+            notes='Burned 0 ETH for gas',
+            counterparty=CPT_GAS,
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=307,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=EvmToken('eip155:1/erc20:0xcd9D82d33bd737De215cDac57FE2F7f04DF77FE0'),
+            balance=Balance(amount=FVal(100000)),
+            location_label='0x00000000000Cd56832cE5dfBcBFf02e7eC639BC9',
+            notes='Receive 100000 variableDebtREN from 0x0000000000000000000000000000000000000000 to 0x00000000000Cd56832cE5dfBcBFf02e7eC639BC9',  # noqa: E501
+            counterparty='aave-v2',
+            identifier=None,
+            extra_data=None,
+        ), HistoryBaseEntry(
+            event_identifier=evmhash,
+            sequence_index=310,
+            timestamp=0,
+            location=Location.BLOCKCHAIN,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_REN,
+            balance=Balance(amount=FVal(100000)),
+            location_label='0x00000000000Cd56832cE5dfBcBFf02e7eC639BC9',
+            notes='Borrow 100000 eip155:1/erc20:0x408e41876cCCDC0F92210600ef50372656052a38 from AAVE V2 with variable APY 0.8024003551450866705556542',  # noqa: E501
```

Comment:
Since the APY is variable I believe it doesn't make much sense to put it here? When is fixed 100% should be but it being variable might be not that useful. Have you tough about that?